Sources/Penny/Handlers/MessageDeleteHandler.swift

```diff
@@ -25,19 +25,22 @@ struct MessageDeleteHandler {
             channelId: channelId
         ) else {
             logger.warning("Could not find a saved message for deleted message", metadata: [
-                "messageId": .stringConvertible(messageId),
-                "channelId": .stringConvertible(channelId)
+                "messageId": .string(messageId.rawValue),
+                "channelId": .string(channelId.rawValue)
```

Comment:
Currently this shows up as e.g. `Snowflake<DiscordChannel>("1226220660427329748")` but after this change it'll be `1226220660427329748`.